packages/core-data/src/resolvers.js

```diff
@@ -976,3 +976,29 @@ export const getRevision =
 			dispatch.receiveRevisions( kind, name, recordKey, record, query );
 		}
 	};
+
+/**
+ * Requests a specific post type options from the REST API.
+ *
+ * @param {string} postType Post type slug.
+ */
+export const getRegisteredPostMeta =
+	( postType ) =>
+	async ( { select, dispatch } ) => {
+		try {
+			const {
+				rest_namespace: restNamespace = 'wp/v2',
+				rest_base: restBase,
+			} = select.getPostType( postType ) || {};
```

Comment:
```suggestion
	async ( { dispatch, resolveSelect } ) => {
		try {
			const {
				rest_base: restBase,
				rest_namespace: restNamespace = 'wp/v2',
			} = await resolveSelect.getPostType( postType );
```

We should probably use `resolveSelect` to await for the post type as done in `getAutosaves`: [link](https://github.com/WordPress/gutenberg/blob/bc6fae8c8d41763b2786fce8f228d96eb89b014c/packages/core-data/src/resolvers.js#L540-L541).

Without this, there is a bug reported [here](https://github.com/WordPress/gutenberg/pull/64072#pullrequestreview-2278752560). If you load a template without bindings and click on a paragraph, it doesn't render the panel.

Reply:
I was just having that issue.

Reply:
Done